File: year2/semester2/mpp/java/TripClientServer/TripCommon/src/main/java/dto/AccountDto.java
package dto;

import java.io.Serializable;

public class AccountDto implements Serializable {
    public String name;
    public String password;
    public final Integer id;

    public AccountDto(String name, String password) {
        this(null, name, password);
    }

    public AccountDto(Integer id, String name, String password) {
        this.id = id;
        this.name = name;
        this.password = password;
    }

    public AccountDto(Integer accountId) {
        id = accountId;
    }
}

File: year2/semester2/mpp/c#/TripClientServer/TripCommon/Dto/AccountDto.cs

using System;

namespace TripCommon.Dto
{
    [Serializable]
    public class AccountDto
    {
        public int id;
        public string name;
        public string password;

        public AccountDto(int accountId)
        {
            this.id = accountId;
        }

        public AccountDto(string name, string password)
        {
            this.name = name;
            this.password = password;
        }

        public AccountDto(int id, string name, string password)
        {
            this.id = id;
            this.name = name;
            this.password = password;
        }

        public override string ToString()
        {
            return "AccountDto{ id=" + id + ", name=" + name + ", password=" + password + "}";
        }
    }
}
